src/db.c

```diff
@@ -1031,11 +1031,12 @@ void moveCommand(client *c) {
     src = c->db;
     srcid = c->db->id;
 
-    if (getLongLongFromObject(c->argv[2],&dbid) == C_ERR ||
-        dbid < INT_MIN || dbid > INT_MAX ||
-        selectDb(c,dbid) == C_ERR)
+    if (getIntFromObjectOrReply(c, c->argv[2], &dbid, NULL) != C_OK)
+        return;
+
+    if (selectDb(c,dbid) == C_ERR)
     {
-        addReply(c,shared.outofrangeerr);
+        addReplyError(c,"DB index is out of range");
```

Comment:
If there is only one line of 'if' judgement. we should append '{' at the tail of this line instead of new line.

For error message, should we keep the same to "invalid DB index"?

There are similar problems below

Reply:
Since selectCommand uses "DB index is out of range", I want to unify the error logs.

Reply:
Currently, selectCommand uses "invalid DB index"

Reply:
> If there is only one line of 'if' judgement and the length is not more than 80 obviously. we should append '{' at the tail of this line instead of new line.

Thanks, It should be exactly as you say.